tests/foreman/ui/test_activationkey.py

```diff
@@ -1384,7 +1384,7 @@ def test_multiple_activation_keys_to_system(self):
             )
             self.assertEqual(
                 result.return_code, 0,
-                "failed to fetch katello-ca rpm: {0}, return code: {1}"
+                u"failed to fetch katello-ca rpm: {0}, return code: {1}"
```

Comment:
We are safe to drop the assert message, we already have these information on robottelo log.